code/modules/clothing/outfits/ert.dm

```diff
@@ -428,6 +430,7 @@
 /datum/outfit/death_commando/officer
 	name = "Death Commando Officer"
 	head = /obj/item/clothing/head/helmet/space/beret
+	backpack_contents = list(/obj/item/aiModule/core/full/deathsquad)
```

Comment:
Overrides the backpack contents of /datum/outfit/death_commando in turn giving them nothing except this

Reply:
Fixed in c70d29c